Source/Plugins/Core/com.equella.core/js/__tests__/tsrc/search/components/SearchResult.test.tsx

```diff
@@ -29,7 +29,11 @@ describe("<SearchResult/>", () => {
     return render(
       //This needs to be wrapped inside a BrowserRouter, to prevent an `Invariant failed: You should not use <Link> outside a <Router>` error  because of the <Link/> tag within SearchResult
       <BrowserRouter>
-        <SearchResult {...itemResult} key={itemResult.uuid} />
+        <SearchResult
+          key={itemResult.uuid}
+          item={{ ...itemResult }}
```

Comment:
If you are using spread here with no modifiers, surely you can just use it directly? 
```suggestion
          item={ itemResult }
```

Reply:
hmmmm... :thinking: 

I originally saw this as a copy of the object, but it was more so spreading it so that it's populate the old props. So yes, with the new props I could happily just drop it in. I'll tweak it.